umi_tools/umi_methods.py

```diff
@@ -1394,6 +1406,47 @@ def __call__(self, inreads):
                 yield self.reads_dict[p][k], k, "bundle"
 
 
+def get_gene_count_tab(infile,
+                       umi_getter=None):
+
+    ''' Yields the counts per umi for each gene
+
+    ignore_umi: don't include the umi in the dict key
```

Comment:
Yes, `ignore_umi` is not a required method. `skip_regex` would be useful but not currently available. I've updated documentation to mark `skip_regex` as "TODO". Thanks!